packages/pdf-service-api/src/lib/healthchecks.js

```diff
@@ -0,0 +1,13 @@
+const { healthcheck } = require('@pins/common');
+const config = require('./config');
+const logger = require('./logger');
+
+const tasks = [{}];
```

Comment:
The object needs to be removed, so reads `[]` - I'm going to fix this as part of the Helm stuff and then you can rebase

Reply:
I can see you have fixed this. Thanks.